examples/docker-helloworld.yaml

```diff
@@ -30,6 +30,7 @@ gateway:
     - controller
     - registry
   container_name: gateway
+  command: /usr/bin/run_filebeat.sh
```

Comment:
this line is always needed - because we do not run filebeat by default.  The ENTRYPOINT for `sidecar:alpine` image is :
```ENTRYPOINT ["/usr/bin/a8sidecar"]```  
I thought the decision was not to run filebeat by default